src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const property of action.keysToRemove) {
+          if (property in state) {
+            delete state[property];
+          }
+        }
+        break;
+
+      case 'clear':
+        for (const property in state) {
+          delete state[property];
+        }
+        break;
```

Comment:
you forgot about default case
![image](https://github.com/mate-academy/js_stateful-object/assets/55200530/cf58fb45-7f18-46a6-90fd-49ae4b647577)
throw error for `default`